File: angular/src/app/hotel/models/hotel-type-dto.ts


import { EntityDto } from '@abp/ng.core';

export class HotelTypeDto extends EntityDto<string> {
  name: string;
  description: string;
  hotel: any;
  id: string;

  constructor(initialValues: Partial<HotelTypeDto> = {}) {
    super(initialValues);
  }
}

File: aspnet-core/src/HotelApp.Application.Contracts/Hotels/HotelTypeDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace HotelApp.Hotels
{
    public class HotelTypeDto : EntityDto<Guid>
    {
        public string Name { get; set; }
        public string Description { get; set; }

        public HotelDto Hotel { get; set; }
    }



}
